book/version_control.md

```diff
@@ -246,107 +241,44 @@ sandpit/
-
 * Under Workspace, select `Never` for `Save workspace to .RData on exit:`.
 * Under History, deselect `Always save history`.
 
 
-#### Jupyter Notebooks
+### Jupyter Notebooks
 
-By default, Jupyter Notebooks save a copy of the data that is used to create cell outputs.
-For example, the data used to generate a chart, table or print a section of a dataframe.
-This can be useful for sharing your code outputs without others needing to re-execute the code cells.
+By default, Jupyter Notebooks save a copy of the data that is used to create cell outputs. For example, the data used to generate a chart, table or print a section of a dataframe. This can be useful for sharing your code outputs without others needing to re-execute the code cells.
 
-If you work with sensitive datasets in notebooks, this means that your notebooks may store sensitive data to display cell outputs.
-If these notebooks are subsequently shared in code repositories, you may be making sensitive data available to unauthorised individuals.
+If you work with sensitive datasets in notebooks, this means that your notebooks may store sensitive data to display cell outputs. If these notebooks are subsequently shared in code repositories, you may be making sensitive data available to unauthorised individuals.
 
 It is [not currently possible to prevent the notebooks from retaining cell outputs](https://github.com/ipython/ipython/issues/1280).
 
-The best way to handle this situation is to clear the outputs from your notebooks before commiting them to Git repositories.
-This can be done from the notebook itself, by going to the menu `Cell > All > Output > Clear` and then saving your notebook.
-Alternatively, this can be done from the command line, by running this command with your notebook file path:
+The best way to handle this situation is to clear the outputs from your notebooks before commiting them to Git repositories. This can be done from the notebook itself, by going to the menu `Cell > All > Output > Clear` and then saving your notebook. Alternatively, this can be done from the command line, by running this command with your notebook file path:
 
 ```
 jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace <notebook_file_path>
 ```
 
-This approach is quite laborious and could easily be forgotten.
-As such, we advise you to automate the cell clearing using [Git filter](https://git-scm.com/docs/gitattributes/#_filter) as follows.
-First, we tell Git to set an attribute on our notebook files by creating a `.gitattributes` file in the root of our repo:
+This approach is quite laborious and could easily be forgotten. As such, we advise you to automate the cell clearing using [Git filter](https://git-scm.com/docs/gitattributes/#_filter) as follows. First, we tell Git to set an attribute on our notebook files by creating a `.gitattributes` file in the root of our repo:
 
 ```
 *.ipynb filter=jupyternotebook
```

Comment:
Wait, why how is the filter `jupyternotebook` relate to the command required to clean it. Where does the aformentioned nbonvert thing come into play as part of this attributes thing.

Reply:
`jupyternotebook` is the name of the filter for the notebook files. The filter command then filters for these files and runs the `clean` command, which is our `nbconvert`. This is a really long-winded way of sorting it, so I'd be keen to replace it with an easier alternative.

I think there's a different approach taken in govcookiecutter that we could use